common/beerocks/bwl/dummy/mon_wlan_hal_dummy.cpp

```diff
@@ -167,6 +167,12 @@ bool mon_wlan_hal_dummy::channel_scan_dump_results()
     return false;
 }
 
+bool mon_wlan_hal_dummy::generate_connected_clients_events()
+{
+    LOG(TRACE) << __func__ << " - NOT IMPLEMENTED";
```

Comment:
I don't think it will never be implemented on dummy mode, IMO you could remove it.

Reply:
Removed